README.md

```diff
@@ -41,6 +57,27 @@ If you want to find a record using it's slug, a custom finder is provided by the
 $example = $this->Examples->find('slugged', ['slug' => $slug]);
 ```
 
+## Configuration
+
+Slug comes with the following configuration options:
+
+- ``field``: name of the field (column) to hold the slug. Defaults to ``slug``.
```

Comment:
Someone told me using double backticks would be the wisest default since everything in between them becomes a literal. Sort of a discussion about using ``'`` or ``"`` but hey... I will change it happily. Last confirmation please?